src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java

```diff
@@ -1056,5 +1059,36 @@ private boolean getMoreAvailable(Page<Organisation> pageableOrganisations) {
         return !pageableOrganisations.isLast();
     }
 
+    @Override
+    public ResponseEntity<ContactInformationResponse> updateContactInformationForOrganisation(
+        ContactInformationCreationRequest contactInfo, String organisationIdentifier) {
+
+        Organisation organisation = organisationRepository.findByOrganisationIdentifier(organisationIdentifier);
+        List<ContactInformation> existingContactInformationList = organisation.getContactInformation();
+        //if multiple contact informations are found they will all be updated
+        if (contactInfo != null) {
+            existingContactInformationList.forEach(existingInfo -> {
+                existingInfo.setAddressLine1(RefDataUtil.removeEmptySpaces(contactInfo.getAddressLine1()));
+                existingInfo.setAddressLine2(RefDataUtil.removeEmptySpaces(contactInfo.getAddressLine2()));
+                existingInfo.setAddressLine3(RefDataUtil.removeEmptySpaces(contactInfo.getAddressLine3()));
+                existingInfo.setTownCity(RefDataUtil.removeEmptySpaces(contactInfo.getTownCity()));
+                existingInfo.setCounty(RefDataUtil.removeEmptySpaces(contactInfo.getCounty()));
+                existingInfo.setCountry(RefDataUtil.removeEmptySpaces(contactInfo.getCountry()));
+                existingInfo.setPostCode(RefDataUtil.removeEmptySpaces(contactInfo.getPostCode()));
+                existingInfo.setOrganisation(organisation);
+                existingInfo.setLastUpdated(LocalDateTime.now());
+                ContactInformation savedContactInformation = contactInformationRepository.save(existingInfo);
+                if (savedContactInformation == null || savedContactInformation.getId() == null) {
```

Comment:
unnecessary null check: saveContactInformation will never be null as per 'CrudRepository.save' method contract. In case passed existingInfo is null it would fail on line 1071 with NPE